showkase-processor-testing/src/test/java/com/airbnb/android/showkase_processor_testing/ShowkaseProcessorTest.kt

```diff
@@ -143,6 +143,11 @@ class ShowkaseProcessorTest : BaseProcessorTest() {
                 "annotation")
     }
 
+    @Test
+    fun `composable previews with parameter providers should indent properly`() {
```

Comment:
@wang-li Can you rename this test to the following since it describes it better. You will also need to rename the corresponding folder.

`composable previews with multiple parameter providers should indent properly`

Reply:
@vinaygaba For sure 👍 It's done in an additional commit 😉